src/Contexts/NotesContext.js

```diff
@@ -2,21 +2,28 @@ import { createContext, useReducer } from "react";
 
 export const NotesContext = createContext();
 
+export const notesActionTypes = {
+  GET_ALL_NOTES: "GET_ALL_NOTES",
+  POST_SINGLE_NOTE: "POST_SINGLE_NOTE",
+  UPDATE_SINGLE_NOTE: "UPDATE_SINGLE_NOTE",
+  DELETE_SINGLE_NOTE: "DELETE_SINGLE_NOTE",
+};
```

Comment:
yes thats what I meant, looks good!

Reply:
Make sure that you use the same dictionary for declaring the action type when you dispatch.